### PR TITLE
chore(squidnlb: changes to the updatewhitelish.sh and commons code  t…

### PR DESCRIPTION
**Pull request by rarya**
…o add the vpc endpoint config.tvars variable

### CODE REVIEW

File: flavors/squid_nlb_central/updatewhitelist.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ cd /home/ubuntu/cloud-automation
 git pull
 
 echo "Checking if list of authorized keys have changed for admin"
-diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/authorized_keys_admin" "/home/ubuntu/.ssh/authorized_keys"
+sudo diff "/home/ubuntu/cloud-automation/flavors/squid_nlb_central/authorized_keys_admin" "/home/ubuntu/.ssh/authorized_keys"
```
Comment (frickjack) [inline flavors/squid_nlb_central/updatewhitelist.sh]: Why do you need `sudo` here, but not in the rsync?

File: gen3/lib/aws.sh [hunk 2]
```diff
@@ -194,7 +194,7 @@ vpc_name="$GEN3_WORKSPACE"
 #
 vpc_octet2=GET_A_UNIQUE_VPC_172_OCTET2
 vpc_octet3=GET_A_UNIQUE_VPC_172_OCTET3
-squid-nlb-endpointservice-name=CSOC_SQUID_NLB_SERVICE_NAME
+squid-nlb-endpointservice-name="CSOC_SQUID_NLB_SERVICE_NAME"
```
Comment (frickjack) [inline gen3/lib/aws.sh]: This hides the variable default in `variables.tf` - put the "com.amazonaws.vpce.us-east-1.vpce-svc-0ce2261f708539011" value here too, or just don't put the variable in `config.tfvars`

File: flavors/squid_nlb_central/updatewhitelist.sh [hunk 3]
```diff
@@ -4,7 +4,7 @@ cd /home/ubuntu/cloud-automation
 git pull
 
 echo "Checking if list of authorized keys have changed for admin"
```
Comment (frickjack) [inline flavors/squid_nlb_central/updatewhitelist.sh]: Does this script run as the `ubuntu` user in a crontab?  If so - please add a comment explaining that.

File: flavors/squid_nlb_central/updatewhitelist.sh [hunk 4]
```diff
@@ -22,23 +22,23 @@ fi
```
Comment (frickjack) [inline flavors/squid_nlb_central/updatewhitelist.sh]: If this script runs as the `ubuntu` user, then I think these `rsync` commands need to be run with `sudo` too

File: flavors/squid_nlb_central/updatewhitelist.sh (hunk 3 above)
Comment (rarya) [reply flavors/squid_nlb_central/updatewhitelist.sh]: the cronjob is run as root; so removed sudo

File: gen3/lib/aws.sh (hunk 2 above)
Comment (rarya) [reply gen3/lib/aws.sh]: I removed it from the config.tvars

**Pull request merged**